mempool/clist_mempool_test.go

```diff
@@ -289,7 +289,7 @@ func TestMempoolUpdateDoesNotPanicWhenApplicationMissedTx(t *testing.T) {
 		reqRes := newReqRes(tx, abci.CodeTypeOK, abci.CHECK_TX_TYPE_CHECK)
 		callback(reqRes.Request, reqRes.Response)
 	}
-	require.Equal(t, len(txs), mp.Size())
```

Comment:
Is that the result of some linting? Neat.

Reply:
Yep